airbyte-scheduler/app/src/main/java/io/airbyte/scheduler/app/SchedulerApp.java

```diff
@@ -211,6 +211,7 @@ public static void main(final String[] args) throws IOException, InterruptedExce
     LOGGER.info("temporalHost = " + temporalHost);
 
     // Wait for the server to initialize the database and run migration
+    // This should be converted into check for the migration version. Everything else as per.
```

Comment:
how will this work in the docker compose case? doesn't it need to wait?

Reply:
Yeap! My current thought is to loop for a time period while checking to see if the migration has been applied. This should be needed for both docker and kube. Will do so in the follow up PR.